cross/hoverkite-firmware/src/hoverboard.rs

```diff
@@ -150,12 +153,50 @@ fn DMA_CHANNEL0() {
     });
 }
 
+/// The buzzer on the secondary board. This should not be used on the primary board.
+pub struct Buzzer {
+    pwm: pwm::Pwm<
+        TIMER1,
+        // This type is a bit bogus, PB10 is the only one that matters.
+        (
+            Option<PA0<Alternate<AF2>>>,
+            Option<PA1<Alternate<AF2>>>,
+            Option<PB10<Alternate<AF2>>>,
```

Comment:
could/should the pwm module export a second type for single-pin pwm that's simpler?